sdk/remoterendering/mixedreality-remoterendering/README.md

```diff
@@ -0,0 +1,408 @@
+its maximum lease time.
+This example shows how to query the current properties and then extend the lease if it will expire soon.
+
+> NOTE: The runtime SDKs also offer this functionality, and in many typical scenarios, you would use them to
+> extend the session lease.
+
+```typescript Snippet:UpdateSession
+/// When the lease is within 2 minutes of expiring, extend it by 15 minutes.
+let currentSession = await client.getSession(sessionId);
+if (currentSession.status == "Ready") {
+  if (
+    currentSession.maxLeaseTimeInMinutes -
+      (Date.now() - currentSession.properties.createdOn.valueOf()) / 60000 <
+    2
+  ) {
+    let newLeaseTime = currentSession.maxLeaseTimeInMinutes + 15;
+
+    await client.updateSession(sessionId, { maxLeaseTimeInMinutes: newLeaseTime });
+  }
+}
+```
+
+### List sessions
+
+You can get information about your sessions using the `getSessions` method.
+This method may return sessions which have yet to start and sessions which are ready.
+
+```typescript Snippet:ListSessions
+for await (const session of client.listSessions()) {
+  if (session.status === "Starting") {
+    console.log(`Session ${session.sessionId} is starting`);
+  } else if (session.status === "Ready") {
+    console.log(`Session ${session.sessionId} is ready`);
+  }
+}
+```
+
+### Stop a session
+
+The following code will stop a running session with given id.
+
+```typescript Snippet:StopSession
+client.endSession(sessionId);
+```
+
+## Troubleshooting
+
+### Enable logs
+
+You can set the following environment variable to see debug logs when using this library.
+
+- Getting debug logs from the Azure TextAnalytics client library
+
+```bash
+export DEBUG=azure*
+```
+
+### Azure Remote Rendering troubleshooting
+
+For general troubleshooting advice concerning Azure Remote Rendering, see [the Troubleshoot page](https://docs.microsoft.com/azure/remote-rendering/resources/troubleshoot) for remote rendering at docs.microsoft.com.
+
+The client methods will throw exceptions if the request cannot be made.
+However, in the case of both conversions and sessions, the requests can succeed but the requested operation may not be successful.
+In this case, no exception will be thrown, but the returned objects can be inspected to understand what happened.
+
+If the asset in a conversion is invalid, the conversion operation will return an AssetConversion object
+with a Failed status and carrying a RemoteRenderingServiceError with details.
+Once the conversion service is able to process the file, a &lt;assetName&gt;.result.json file will be written to the output container.
+If the input asset is invalid, then that file will contain a more detailed description of the problem.
+
+Similarly, sometimes when a session is requested, the session ends up in an error state.
+The startSessionOperation method will return a RenderingSession object, but that object will have an Error status and carry a
+RemoteRenderingServiceError with details.
```

Comment:
Please add a logging section similar to the one in https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/textanalytics/ai-text-analytics/README.md:
```suggestion
RemoteRenderingServiceError with details.

### Logging

Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:

```javascript
import { setLogLevel } from "@azure/logger";

setLogLevel("info");
```

For more detailed instructions on how to enable logs, you can look at the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/logger).
```

Reply:
Agh, gh rendering of the suggestion is broken!